### PR TITLE
docs: add tags as categories

### PR DESCRIPTION
**Pull request by debs-obrien**

### CODE REVIEW

File: docs/src/troubleshooting.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 id: troubleshooting
 title: "Troubleshooting"
+tags:
+  - Devops
```
Comment (mxschmitt) [inline docs/src/troubleshooting.md:5]:
```suggestion
  - DevOps
```

File: docs/src/selenium-grid.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 ---
 id: selenium-grid
 title: "Selenium Grid"
+tags:
+  - Devops
```
Comment (mxschmitt) [inline docs/src/selenium-grid.md:5]:
```suggestion
  - DevOps
```

File: docs/src/ci-intro-js.md [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
 **What you will learn:**
 
-- [How to use GitHub Actions to run your tests](#github-actions)
-- [How to create a repo and push to GitHub](#create-a-repo-and-push-to-github)
-- [How to open the workflows](#opening-the-workflows)
-- [How to view the test logs](#viewing-test-logs)
-- [How to download the report from GitHub](#downloading-the-playwright-report)
-- [How to view the report](#viewing-the-playwright-report)
-- [How to view the trace](#viewing-the-trace)
+- [How to use GitHub Actions to run your tests](/intro.md#github-actions)
```
Comment (mxschmitt) [inline docs/src/ci-intro-js.md:13]: whats the reason for that change?

File: docs/src/ci-intro-js.md [hunk 4]
```diff
@@ -1,19 +1,22 @@
 ---
 id: ci-intro
 title: "CI Github Actions"
+tags:
+  - Getting started
+  - Devops
```
Comment (mxschmitt) [inline docs/src/ci-intro-js.md:6]:
```suggestion
  - DevOps
```

File: docs/src/browsers.md [hunk 5]
```diff
@@ -1,6 +1,8 @@
 ---
 id: browsers
 title: "Browsers"
+tags:
+  - Devops
```
Comment (mxschmitt) [inline docs/src/browsers.md:5]:
```suggestion
  - DevOps
```

File: docs/src/ci-intro-js.md (hunk 3 above)
Comment (debs-obrien) [reply docs/src/ci-intro-js.md:13]: if you don't do this docusauras will treat it as a TOC and add the whats next section and update it so the only way to trick it is to add that change.

**Pull request closed without merging**